src/components/ProgressBar.scss

```diff
@@ -0,0 +1,81 @@
+// @import "../base/index.scss";
+
+//------------------------------------------------------------//
+
+// * Progress Bar
+
+// TODO : "completed" status originally appeared in both the
+//        positive group and the neutral group
+
+//------------------------------------------------------------//
+
+.ProgressBar {
+}
+
+.ProgressBar-title {
+  margin: 0;
+  padding-top: $su-small;
+  padding-bottom: $su-xsmall;
+  text-align: left;
+  .ProgressBar--small & {
+    font-size: $tu-small-fontSize;
+  }
+  .ProgressBar--large & {
+    font-size: $tu-large-fontSize;
+  }
+}
+
+.ProgressBar-range {
+  position: relative;
+  background: $cu-background;
+  height: $su-small;
+  border-radius: ($su-small / 2);
+  margin-top: $su-xsmall;
+  margin-bottom: $su-default;
+  box-shadow: 0 0 2px darken($cu-background, 3%) inset;
```

Comment:
This can be updated to use $inset-box-shadow-medium and $inset-box-shadow-small for the smaller progress bars